packages/storefront-sdk/codegen.yml

```diff
@@ -0,0 +1,10 @@
+overwrite: true
+schema: 'https://storefront.api.nacelle.com/graphql/v1/spaces/${NACELLE_SPACE_ID}'
+documents: 'src/graphql/**/*.ts'
+generates:
+  src/types/storefront.ts:
+    plugins:
+      - typescript
+      - typescript-operations
+    config:
+      enumsAsTypes: true
```

Comment:
If you're wondering why we're using `codegen.yml` instead of `codegen.ts`, despite what [the GraphQL Code Generator docs](https://the-guild.dev/graphql/codegen/docs/getting-started/esm-typescript-usage) say, there is a bug that prevents us from using `codegen.ts` in an ESM project: https://github.com/dotansimha/graphql-code-generator/issues/8488.